### PR TITLE
Added Division Feature

### PR DESCRIPTION
**Pull request by Santanu-99**
Hey, I have added a Division.java file and Division feature in Calculator.java. Please check it out !!!

### CODE REVIEW

**Comment by R25bitzcode** [comment]
hey, Great Collaboration.

File: Division.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Division{
+    int divide(int n1,int n2){
+        return n1/n2;
```
Comment (R25bitzcode) [inline Division.java]: Hey, you have not checked for Division by Zero Error

Comment (R25bitzcode) [reply Division.java]: is this reaching you??

**Comment by R25bitzcode** [comment]
Is this even getting to you???

File: Division.java (hunk 1 above)
Comment (Santanu-99) [reply Division.java]: Yeah, it has reached me. I will update the code as requested

Comment (Santanu-99) [reply Division.java]: Hey, I have updated the code please check it out

Comment (R25bitzcode) [reply Division.java]: Okay. It's perfect.

**Pull request merged**